x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_exit_modal_message.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiSpacer, EuiText } from '@elastic/eui';
+import type { HostMetadata } from '../../../../common/endpoint/types';
+import {
+  ConsoleExitModalActionLogLink,
+  HostNameText,
+} from '../console/components/console_manager/components/console_exit_modal_link';
```

Comment:
ahhhh. Ok. I see it - its used here. Please move that component (and i18n key) to be inside of `endpoint_responder/` folder.

Reply:
done c6ab1c82cfcbd5f90ef2bf36e51d7990c7de7cc5